temporian/core/operators/base.py

```diff
@@ -61,6 +61,7 @@ class Operator(ABC):
   def __init__(self):
     self._inputs = {}
     self._outputs = {}
+    self._attributes = {}
```

Comment:
Can the attributes be typed to the allowed types (str and int) ? (https://github.com/google/temporian/blob/main/temporian/proto/core.proto#L60)

Reply:
Adding type hint `self._attributes: dict[str, Union[str, int]] = {}`, if you referred to something else lmk 🙌🏼

Reply:
Looks good.